src/patternfly/components/Chip/examples/chip-complex-example.hbs

```diff
@@ -0,0 +1,128 @@
+{{#> chip-group chip-group--attribute='role="group of chips"'}}
+  {{#> chip}}
+    {{#> chip-text}}
+      Value One
+    {{/chip-text}}
+    {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute='aria-label="Remove"'}}
```

Comment:
Currently, all Remove buttons have the same label, which can be really confusing for screen reader users. 

We can include `aria-labelledby` to fix that:
```
  <div class="pf-c-chip">
    <p class="pf-c-chip__text" id="chip_one">
      Value One
    </p>
    <button class="pf-c-button pf-m-plain" aria-labelledby="remove_chip_one chip_one" aria-label="Remove" id="remove_chip_one">
      <i class="fas fa-times-circle" aria-hidden="true"></i>
    </button>
  </div>
```

The attribute `aria-labelledby` takes one or more values of `id` attributes that are defined. This solution requires an `id` to be added to both the chip text and the remove button. This will result in the labels of both elements being announced when the Remove button receives focus. So the result should be something like: "Remove Value One button"